tests/python/test_internal_func.py

```diff
@@ -50,3 +50,11 @@ def test():
 
     test()
     test()
+
+
+def _test_node_manager_gc():
```

Comment:
Why does it begin with `_`?

Reply:
The bug is not fixed, so running this test will actually fail.

Reply:
I added the fix now